### ISSUE TITLE
Did the new layout broke it ?

### ISSUE THREAD

**Issue by redbaty** [gigobyte/HLTV#4]
It doesn't seem to be working after the layout update, even on runkit

**Comment by gigobyte** [comment]
Yes, the new HLTV design completely broke this parser, there is a fix from @LaCTuK that I'm going to publish as version 0.2.2, but there is going to be a major version bump soon when I finish rewriting the library.

**Comment by gigobyte** [comment]
As of version 1.0.0 the issue is now fixed.

**Comment by redbaty** [comment]
Oh wow, that was fast, great work ! 👍
